spec/models/project_spec.rb

```diff
@@ -83,7 +83,8 @@
   end
 
   describe '#related_projects' do
-    let(:organization) { Organization.create!(name: 'CodeMontage') }
+    let(:organization) { Organization.create!(name: 'CodeMontage', 
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.